source/test/TestApp/TestUtils.cpp

```diff
@@ -3,8 +3,44 @@
 #include "TestAppPch.h"
 #include "TestUtils.h"
 
+#include <random>
+#include <sstream>
+
 namespace PlayFabUnit
 {
+    
+    std::string GenerateUuidV4() {
```

Comment:
I think that "TestUtils" is a big improvement, but I'll note that the main SDK and thus most customers still won't have direct access to this.
I think I would have been happier with a generic PlayFabUtil.GenerateUuidV4.